pages/roadmap.tsx

```diff
@@ -1,7 +1,183 @@
+  const sectionOneText =
+    "*At the beginning of any journey, one must set a plan of where they are going and how they are going to get there. Defining a destination, establishing a team, and navigating a path, is essential to success.*"
+  const sectionOneList = ["Define Mission/Vision", "Build Roadmap", "Build Whitepaper", "Brand Identity"]
+
+  const sectionTwoTitle = `The Journey`
+  const sectionTwoText =
+    "The mark of our *lineage is the NFT collection, highlighting the ancestral journey. The community’s directive to mint an ancestor will serve not only to the past, but ownership in it and the future. Diaspora will build mechanisms to tell this story of lineage and the journey along the way.*"
+  const sectionTwolist = ["Launch NFT Collection", "Build Gallery", "Marketing/Social"]
+
+  const sectionThreeTitle = `Mint an Ancestor`
+  const sectionThreeText =
+    "*Culture is “the art and other manifestations of human intellectual achievement regarded collectively” - Oxford Languages. In this phase we will build the infrastructure of the DAO to allow for organic development of the community at scale.*"
+  const sectionThreelist = ["Develop DAO", "Develop Structure and Governance", "Logistics (LLC, DeFi, etc.)"]
+
+  const sectionFourTitle = `Lay the Cultural Foundation`
+  const sectionFourText =
+    "*Through this non linear journey, we will build expansive omni directional paths for members to take. Building on the culture/pillars, members can choose their journey and participate in events and initiatives that will foster the Diaspora community in physical and metaverse form.*"
+  const sectionFourlist = [
+    "Building Partnerships (in the 4 pillars)",
+    "Develop Event Schedule and Framework (Physical and Digital)",
+    "Develop Masterclasses",
+  ]
+
+  const sectionFiveTitle = `Building the Pathways`
+
+  const sectionSixTitle = `Institutionalize the Revolution`
+  const sectionSixText = `*Building this DAO is both a digital and physical revolutionary act. In order to sustain it, future facing, mechanisms have to be built in place to future proof.*`
+  const sectionSixlist = ["Buy Land in Metaverse", "Launch the Diaspora Token"]
+
+  // ui components
+  return (
+    <div className="roadmap-background">
+      <div className="roadmap-pagnation">
+        <p
+          className={`${activeIndex === 1 ? "roadmap-pagnation-dot-active" : "roadmap-pagnation-dot-unactive"}`}
+          onClick={() => {
+            setActiveIndex(1)
+            !!sectionOne.current && sectionOne.current.scrollIntoView()
+          }}
+        />
+        <p
+          className={`${activeIndex === 2 ? "roadmap-pagnation-dot-active" : "roadmap-pagnation-dot-unactive"}`}
+          onClick={() => {
+            setActiveIndex(2)
+            !!sectionTwo.current && sectionTwo.current.scrollIntoView()
+          }}
+        />
+        <p
```

Comment:
I would make an array of sections and use a .map to iterate over it and render the sections since the code is essentially the same